test/integration/mysql/mysql_test.go

```diff
@@ -36,6 +37,9 @@ func TestMysql(t *testing.T) {
 		// DefaultVerify asserts no resource changes exist after apply.
 		// It helps ensure that a second "terraform apply" wouldn't result in resource deletions/replacements.
 		blueprintTest.DefaultVerify(assert)
+
+		deploymentUrl := blueprintTest.GetStringOutput("endpoint") // Looks like https://three-tier-app-fe-abcde12ghi-uc.a.run.app
```

Comment:
Cloud Run URLs won't necessarily be this format, with the launch of Deterministic URLs

Reply:
Ah, good catch. Removing the comment (`// Looks like https://three-tier-app-fe-abcde12ghi-uc.a.run.app`).